src/hotspot/share/cds/filemap.cpp

```diff
@@ -195,6 +193,13 @@ FileMapInfo::~FileMapInfo() {
     assert(_dynamic_archive_info == this, "must be singleton"); // not thread safe
     _dynamic_archive_info = NULL;
   }
```

Comment:
Original change adds an empty line here. Please insert.

Reply:
Fixed. Thanks for the review, @TheRealMDoerr